tests/nlu/test_spacy_utils.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+from rasa.nlu.model import InvalidModelError
+from rasa.nlu.config import RasaNLUModelConfig
+from rasa.nlu.utils.spacy_utils import SpacyNLP
+
+
+def test_model_fallback_raises_error():
```

Comment:
@wochinge is there a way to flag these tests that they should be removed in Rasa 3.0?

Reply:
no need in my opinion. The test fails when removing the production code anyway. 
Btw, isn't this test already covered by the test function at the bottom of this module?